src/models/pocketSave.ts

```diff
@@ -0,0 +1,48 @@
+import { PocketSave } from '../types';
+import { IContext } from '../server/context';
+import { ListResult, PocketSaveDataService } from '../dataService';
+import { NotFoundError } from '@pocket-tools/apollo-utils';
+
+export class PocketSaveModel {
```

Comment:
nit: class docs - looking for some details on responsibility of this class with mutation and query workflow

Reply:
so models will be our service layer, and `*Service` will be our dbAccessLayer

Reply:
Ask @kschelonka but yes that is my understanding and what I tried to line out in the PR description.

Reply:
yup, would be good to have it in class docs  :)